test/tools/libtesteth/TestSuite.h

```diff
@@ -19,7 +19,9 @@
  */
 
 #pragma once
+#include <boost/filesystem/path.hpp>
 #include <test/tools/libtesteth/JsonSpiritHeaders.h>
+namespace fs = boost::filesystem;
```

Comment:
Avoid aliases in global namespace in the header files. You can put it inside `namespace dev { namespace test {`